src/components/GoalContentHeader/GoalContentHeader.tsx

```diff
@@ -45,19 +45,19 @@ export const GoalContentHeader: FC<GoalContentHeaderProps> = ({ name, descriptio
         notifyPromise(copyValue(description), 'copy');
     }, [copyValue, description]);
 
-    const dropdownItems = useMemo(
-        () =>
-            navigator?.clipboard
-                ? [
-                      {
-                          label: tr('Copy raw'),
-                          icon: <StyledIconClipboardOutline size="xxs" />,
-                          onClick: onCopyDescription,
-                      },
-                  ]
-                : [],
-        [onCopyDescription],
-    );
+    const dropdownItems = useMemo(() => {
+        if (typeof window === 'undefined') return;
```

Comment:
please make it an util function like `isServer` instead of copying it all around